t/t2212-job-manager-jobspec.t

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+test_description='Checkout job manager redacted jobspec'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 1
+
+RPC=${FLUX_BUILD_DIR}/t/request/rpc
+
+# Usage: job_manager_getattr ID ATTR
+job_manager_getattr() {
+	echo '{"id":'$1',"attrs":["'$2'"]}' | ${RPC} job-manager.getattr
+}
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'stop the queue for this test' '
+	flux queue stop
+'
+
+test_expect_success 'create simple jobspec' '
+	flux mini submit --dry-run hostname >simple.json
+'
+
+test_expect_success HAVE_JQ 'jobspec contains environment' '
+	jq -e -j .attributes.system.environment <simple.json >env.json
```

Comment:
nit: i don't think you need `-j` for these tests, since you're not comparing output

Reply:
OK, fixed that and forced a push.